.github/workflows/just-nut.yml

```diff
@@ -101,5 +101,3 @@ jobs:
         with:
           retry_wait_seconds: 1
           command: ${{ inputs.command }}
-          # TODO: any reason we dont retry on errors AND timeouts? (any)
-          retry_on: error
```

Comment:
Removing this will default to `any` error. We had an example this morning where a NUT did not re-run because of a timeout https://github.com/salesforcecli/cli/actions/runs/10209026168/job/28246609426